transcribed/page1140.txt

```diff
@@ -0,0 +1,26 @@
+001                                                                             PAGE 114
+002     104 002 JDB7A:  MOVE C,SUNIT
+003     103 024         MOVE A,EUPOS(C)
+004     103 022         SUB A,UDIR(C)
+005     103 022         SUB A,UDIR(C)
+006     001 007         MOVEI B,1
+007     103 023         CAML A,UGOAL(C)
+008     001 007         MOVNI B,1
+009     104 003         DPB C,[30300.,SUNITL
```

Comment:
The period should be a comma.  `,,` is MIDAS syntax for separating half words.

Reply:
It sure _looks_ like a period in the photo, but it is slightly out of focus, and there _could_ have been a problem in the original text, but the point of this repo is to get something that compiles! :)